modules/microshift-enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-workload-partitioning.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-enabling-workload-partitioning_{context}"]
+= Enabling workload partitioning
+
+System configuration changes has to be done to enable workload partitioning on {microshift-short}. This can be done by making the following changes:
+* Add Kubelet configuration to enable and configure CPU Manager for the workloads.
```

Comment:
in the preview page this doesn't appear at the bullets.
see:
![image](https://github.com/user-attachments/assets/d3188793-db95-4de7-baf1-1f079bb4b389)

Reply:
this sentence is updated